docs/configuration.md

```diff
@@ -714,6 +714,13 @@ Apart from these, the following properties are also available, and may be useful
 <table class="table">
 <tr><th>Property Name</th><th>Default</th><th>Meaning</th></tr>
 <tr>
+  <td><code>spark.eventLog.blockUpdates</code></td>
```

Comment:
I think it would be better to change the configuration name to be ended with ".enabled", to reflect this is a boolean configuration.